client/src/EstimatesComparison/scheme.js

```diff
@@ -501,12 +501,13 @@ export class EstimatesExplorer extends AbstractExplorerScheme {
       };
     }
     if (type === "column_header_click") {
-      const { is_descending, sort_col } = state;
-      const clicked_col = payload;
+      const { sort_col } = state;
+      const clicked_col = payload.col_key;
+      const is_descending_bool = payload.is_descending_bool;
```

Comment:
Minor note, a name like `is_...` implies a boolean generally (or possibly a getter or, less likely, a method that returns a boolean), or at least something that is truthy/falsey that can be treated as one if we're playing it loose with types. Either way, encoding type in the variable name is usually avoided/avoidable.